frontend/src/scenes/organization/membersLogic.tsx

```diff
@@ -101,6 +102,22 @@ export const membersLogic = kea<membersLogicType>([
                 return updatedMembers
             },
         },
+        scopedApiKeys: {
+            __default: null as OrganizationMemberScopedApiKeysResponse | null,
+            loadMemberScopedApiKeys: async ({ member }: { member: OrganizationMemberType }) => {
+                try {
+                    const res = await api.organizationMembers.scopedApiKeys.list(member.user.uuid)
+                    return res
+                } catch (e) {
+                    // Return a properly structured empty response instead of null
+                    return {
+                        has_keys: false,
+                        keys_active_last_week: false,
+                        keys: [],
+                    }
```

Comment:
Either way the code consuming this has to handle the `null` state, so why not leave `null` here instead of a bogus empty response?

Reply:
Agree, I was getting kea issues. Let me see if I can get it working without this.